src/main/java/com/teamabnormals/savage_and_ravage/core/other/SREvents.java

```diff
@@ -142,14 +142,14 @@ public static void onRightClickEntity(PlayerInteractEvent.EntityInteract event)
 			Player player = event.getEntity();
 			CompoundTag persistentData = target.getPersistentData();
 			if (!persistentData.getBoolean(POISON_TAG)) {
-				if (target.level.random.nextDouble() < SRConfig.COMMON.poisonPotatoChance.get()) {
+				if (target.level.getRandom().nextDouble() < SRConfig.COMMON.poisonPotatoChance.get()) {
 					target.playSound(SoundEvents.GENERIC_EAT, 0.5f, 0.25f);
 					persistentData.putBoolean(POISON_TAG, true);
 					if (SRConfig.COMMON.poisonPotatoEffect.get()) {
 						((LivingEntity) target).addEffect(new MobEffectInstance(MobEffects.POISON, 200));
 					}
 				} else {
-					target.playSound(SoundEvents.GENERIC_EAT, 0.5f, 0.5f + target.level.random.nextFloat() / 2);
```

Comment:
Shouldn't this be using the `RandomSource` instance of the `target` local variable?